src/lib/core.js

```diff
@@ -17,6 +17,8 @@ module.exports = function (options) {
     creds.options.prefix = ledgerPrefix
     core.addClient(ledgerPrefix, new ilpCore.Client(Object.assign({}, creds.options, {
       debugReplyNotifications: config.features.debugReplyNotifications,
+      // disable connect() timeout
+      connectTimeout: 0,
```

Comment:
This suggests that `connectTimeout` should be a standardized parameter in the ledger plugin interface. Is that right?